docker/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# Dockerfile for awesomeWM.
```

Comment:
Yes, but wouldn't another level welcome? I usually see Dockerfile being built with a directory path instead of the Dockerfile absolute path. This allows for image specific assets to be "encapsulated" into the sub directory. Something like:

```sh
$BASE_DIR/docker/docker-awesome_coverage/Dockerfile # The current Alpine base 
$BASE_DIR/docker/docker-awesome_ppa/Dockerfile # Ubuntu LTS with PPA upload (no build)
$BASE_DIR/docker/docker-awesome_dotdeb/Dockerfile # "make package" for Debian stable and testing
$BASE_DIR/docker/docker-awesome_coverity/Dockerfile # Static analysis 
$BASE_DIR/docker/docker-awesome_scanbuild/Dockerfile # Static and dynamic analysis
$BASE_DIR/docker/docker-awesome_bleeding_edge/Dockerfile # Use all the AUR -git / luarocks dependencies to get early breakage warnings.
```

Note that those Dockerfiles can be written over time. I already have a base for the Coverity one (still need to ask for freebies from them. It is usually very expensive, but they it away as long as the code is Free).